### DISCUSSION TITLE
the details for how to get the performance on BTCV leaderboard using Unetr

### DISCUSSION THREAD

**Discussion by sherjy** [Project-MONAI/MONAI#2724] [Q&A]
Hiiii, thanks a lot for your nice framework developing! I noticed you implemented Unetr model and there's a tutorial on https://github.com/Project-MONAI/research-contributions/tree/master/UNETR/BTCV page. I have a question for how to ahieve the SOTA score on leaderboard using this model implementation in monai or following tutorial guidance since I ran the tutorial and it seems only ~0.74 performance achieved on the 6 test samples which is a small test split in tutorial. I haven't submitted to BTCV test server and I wonder if it can achieve ~0.86 performance on 20 test samples using this tutorial implementation. Have you guys tried? Thanks in advance for solving my concerns!!!

**Comment by Nic-Ma** [comment]
Hi @ahatamiz ,

Could you please help share some comments here?

Thanks in advance.
